types/oscal/validation_root/validation_root.go

```diff
@@ -0,0 +1,415 @@
+// Code generated by go generate; DO NOT EDIT.
+package validation_root
+
+import ()
+
+// NOT TO BE USED IN A METASCHEMA
+type VALIDATIONRoot struct {
```

Comment:
@isimluk we should ensure that the name of the structs are converted into camel case ... should be `ValidationRoot`

Reply:
Thanks. I was considering that.

But then ... seeing that struct was ever used, I prioritized other work. Since this is the only badly-named struct **and** it is never used. I was not convinced this was worth.

Note: I plan to keep the generated code untouched, enabling periodic re-generation as OSCAL standard evolves.

If you want me to fix the generation code, I am happy to do that. Just be please be mindful it does hardly brings any value.

Reply:
Fair enough. Can leave this as is.